tests/ui/setup.js

```diff
@@ -0,0 +1,32 @@
+/**
```

Comment:
Is it running properly on Travis? Like headless, phanton process

Reply:
@cassiodias yeah it runs properly on travis